doc/source/whatsnew/v1.5.0.rst

```diff
@@ -83,32 +83,41 @@ did not have the same index as the input.
 
 .. code-block:: ipython
 
+    In [3]: df.groupby('a', dropna=True).transform('sum')
```

Comment:
maybe be worth commenting on each of these *what* is changing (e.g. like you did for [3])